pkg/api/interceptor.go

```diff
@@ -174,9 +176,18 @@ func (wa *WalletAuthorizer) applyLimits(
 	if len(wallet) > 0 && wa.AllowLists {
 		isPriority = wa.AllowLister.IsAllowListed(wallet.String())
 	}
+
 	cost := 1
 	limitType := ratelimiter.DEFAULT
 	switch req := req.(type) {
+	case *mlsv1.RegisterInstallationRequest, *mlsv1.UploadKeyPackageRequest, *identityv1.PublishIdentityUpdateRequest:
```

Comment:
There's a part of me that wants to make publishing identity updates 500X the cost of a normal message. Would kneecap SPAM by forcing them to use fewer accounts that would be easier to block. The main casualty would be E2E tests, since most people register very few accounts.

We can roll that out later if needed.